Phase #3/app/model/Student.cs

```diff
@@ -0,0 +1,13 @@
+public class Student
+{    
+     public List<Grade> Grades {get; }
+     public string FirstName {get; set;}
+     public string LastName {get; set;}
+     public int StudentNumber {get; set;}
```

Comment:
List<Grade> also holds StudentNumber for each lesson so it's a duplicate value.
Possible Solution: Introduce a new Class to only hold values of lesson and score.

Reply:
✔